modules/microshift-greenboot-create-health-check-script.adoc

```diff
@@ -1,57 +1,114 @@
-// Module included in the following assemblies:
+//Updated title and ID:
+//Module included in the following assemblies:
 //
-// * microshift_running applications/microshift-greenboot.adoc
+//* microshift_running-applications/microshift-greenboot-workload-scripts.adoc
 
-:_content-type: PROCEDURE
-[id="microshift-greenboot-create-health-check-script_{context}"]
-= Creating a health check script
+:_content-type: CONCEPT
+[id="microshift-greenboot-app-health-check-script_{context}"]
+= How to create a health check script for your application
 
-You can create a health check script for installed workloads by placing them in the `/etc/greenboot/check/required.d` directory. The following procedure provides an example of installing the busybox application and creating a health check script for busybox. You can use this example as a general guide for creating health check scripts for your applications.
+You can create workload or application health check scripts in the text editor of your choice using the example in this documentation. Then save the scripts in the `/etc/greenboot/check/required.d` directory. When a script in the `/etc/greenboot/check/required.d` directory exits with an error, greenboot triggers a reboot in an attempt to heal the system.
 
-.Prerequisite
+[NOTE]
+====
+Any script placed in the `/etc/greenboot/check/required.d` directory will trigger a reboot if it exits with an error.
+====
 
-* You have installed a workload. For this example, the busybox application is used as a workload. The "Additional resources" section that follows this procedure has a link to instructions on deploying workloads using manifests.
+If your health check logic requires any post-check steps, you can also create additional scripts and save them in the relevant greenboot directories:
```

Comment:
This should all actually be documented in RHEL but they haven't done it yet. It's both ways in the Fedora repo, so I just went lowercase since it's a community name. Arbitrary choice for my convenience, but applied to other section in Install already. Not sure where how they'll do it in RHEL, but I was thinking to revisit once they do and conform to their style.